lib/appsignal/config.rb

```diff
@@ -147,6 +151,10 @@ def config_file
         root_path.nil? ? nil : File.join(root_path, 'config', 'appsignal.yml')
     end
 
+    def detect_from_system
```

Comment:
The name of this method doesn't really describe what it's purpose is. I'd rather use a really long but descriptive method name (like `set_running_in_container_config`?).